HelpSource/Classes/BufCombC.schelp

```diff
@@ -6,12 +6,7 @@ categories::  UGens>Delays>Buffer
 
 Description::
 
-Comb delay line with cubic interpolation which uses a buffer for its
-internal memory. See also  link::Classes/BufCombN::  which uses no
-interpolation, and  link::Classes/BufCombL::  which uses linear
-interpolation. Cubic interpolation is more computationally expensive
-than linear, but more accurate.
-
+UGen consisting of a Comb delay line featuring cubic interpolation, taking advantage of a buffer for internal memory. Check link::Classes/BufCombN::  (using no interpolation), and link::Classes/BufCombL::  (uses linear interpolation). Cubic interpolation is more CPU intensive than linear, yet more accurate (sounds better).
```

Comment:
I personally find this wording less clear. Your note at the end ("(sounds better)") can be a fine addition, though.